mibidata/tiff.py

```diff
@@ -345,6 +344,9 @@ def _convert_from_previous(description):
         warnings.warn(
             'The "folder" attribute is required if "fov_id" is specified. '
             'Setting "folder" to {}.'.format(description['mibi.folder']))
+    if description.get('mibi.aperture'):
+        description['mibi.aperture'] = mi.MibiImage.validate_aperture(
+            description['mibi.aperture'])
```

Comment:
This is much nicer than what I had suggested.

Reply:
👍